Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Tooltips/TooltipEnums/AttachPointType.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.using System.Collections;
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.ToolTips
+{
+    /// <summary>
+    /// Used to find a pivot point that is closest to the 
+    /// anchor. This ensures a natural-looking attachment where the connector line
+    /// meets the label.
+    /// </summary>
+    public enum ToolTipAttachPointType
+    {
+        // Specific options
+        // These double as array positions
+        BotMiddle = 0,
```

Comment:
Per MSDN conventions, please use full words for these values (ex: BottomMiddle instead of BotMiddle).

Reply:
Please add summary tags for api documentation generation.